src/shared/redux/slices/publicationsSlice.ts

```diff
@@ -1,31 +1,58 @@
-import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { createSlice, current, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
The `current` import here can be removed ([eslint](https://github.com/vnLab-Lodz/PubLab/runs/4065171538?check_suite_focus=true#step:4:41)).